doc/xml/firewall-cmd.xml.in

```diff
@@ -2399,225 +2399,93 @@ For interfaces that are not under control of NetworkManager, firewalld tries to
     <refsect2 id="lockdown_options">
```

Comment:
How do you feel about fully removing these sections? I don't think there is any value to have them just to say "this feature has been removed".